src/components/ComboBox/ComboBox.js

```diff
@@ -234,13 +245,12 @@ export default class ComboBox extends React.Component {
                   onKeyDown: this.handleOnInputKeyDown,
                 })}
               />
-              {inputValue &&
-                isOpen && (
```

Comment:
Is the removal of `isOpen` here intentional?

Reply:
Good catch! Thanks @asudoh 🙏

Reply:
Yes, it's intentional. By removing it, we show the clear selection option not only when the ComboBox is opened.